asteroid_filterbanks/enc_dec.py

```diff
@@ -130,15 +131,26 @@ class Encoder(_EncDec):
             If True (default), forwarding input with shape (batch, 1, time)
             will output a tensor of shape (batch, freq, conv_time).
             If False, will output a tensor of shape (batch, 1, freq, conv_time).
-        padding (int): Zero-padding added to both sides of the input.
+        padding (Union[int, List[int]): Zero-padding added to both sides of the input.
+        pad_mode (str): See torch.nn.functional.pad.
 
     """
 
-    def __init__(self, filterbank, is_pinv=False, as_conv1d=True, padding=0):
+    def __init__(
+        self, filterbank, is_pinv=False, as_conv1d=True, center=False, padding=0, pad_mode="reflect"
```

Comment:
By the way, pad_mode=reflect was not the default before, it was constant. But if we want to match torch.stft, they use reflect. To be discussed.